packages/f-header/test-utils/component-objects/f-header.component.js

```diff
@@ -1,5 +1,6 @@
 const loginLink = () => $('[data-test-id="login-link"]');
-const offersLink = () => $('[data-test-id="offers-link]"');
+const offersLinkMobile = () => $('[data-test-id="offers-link-mobile]"');
```

Comment:
This line also needs that change:
```suggestion
const offersLinkMobile = () => $('[data-test-id="offers-link-mobile"]');
```